test/parallel/test-module-create-require.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+// Flags: --expose-internals
+
```

Comment:
👆 Just curious what the `// Flags: --expose-internals` meta comment is about? 
_(In this case I don't see anything internal in the test file)_